Tests/SwiftDocCUtilitiesTests/DirectoryMonitorTests.swift

```diff
@@ -80,15 +80,16 @@ class DirectoryMonitorTests: XCTestCase {
         
         // Wait asynchronously for the expectations to fullfil.
         if expectedChangeOrigin != nil {
-            wait(for: [updated], timeout: 5.0)
+            wait(for: [updated], timeout: 1.5)
         }
         
         if isTreeReloadExpected {
-            wait(for: [reloaded], timeout: 5.0)
+            wait(for: [reloaded], timeout: 1.5)
```

Comment:
These timeouts don't speed up the test as long as it passes. Even if I increase both of these to 100 second `testMonitorUpdates` only takes ~5 seconds for me to run locally.